docs/common/rules_index.ldoc

```diff
@@ -0,0 +1,72 @@
+--<table class='widget_list' border=1>
```

Comment:
Why is this file added to the source if it is generated during the build?

Reply:
dependency loop, see above